packages/pg-pubsub/src/index.ts

```diff
@@ -161,6 +161,16 @@ const plugin: PostGraphilePlugin = {
       let client: pg.PoolClient;
       try {
         client = await pgPool.connect();
+        client.once("error", async e => {
+          // eslint-disable-next-line no-console
+          console.error(`Error occurred in subscriptions client; retrying...`);
+          // eslint-disable-next-line no-console
+          console.error(e);
+          releaseClient(client);
+          if (!pgPool.ending) {
+            setupClient();
+          }
+        });
```

Comment:
This needs to be defined as a function and then `client.on('error', fn)`; then we also need a `client.removeListener('error', fn)` when the client is released or when an error occurs. Otherwise if the client is successfully and cleanly returned to the pool, this error handler will stick around, and it will cause issues.

Reply:
But I use the `once` method - as soon as the event emits, the listener unsubscribes.